### DISCUSSION TITLE
Allow Any IP to connect to Manager

### DISCUSSION THREAD

**Discussion by g33kroid** [Security-Onion-Solutions/securityonion#2091] [Unsupported Versions]
I would like to know how to allow any IP to connect to Manager Web Interface I have tried different CDIR Ranges but with no use

**Comment by weslambert** [comment]
Theoretically, you could run so-allow, with the `a` option specifying 0.0.0.0/0 as the range, but obviously that would not be recommended as best practice.  Did you specify to access the web interface by IP or Hostname, and have you tried checking the logs for services like Nginx to ensure there are no issues there?

**Reply by g33kroid** [reply]
> Theoretically, you could run so-allow, with the `a` option specifying 0.0.0.0/0 as the range, but obviously that would not be recommended as best practice. Did you specify to access the web interface by IP or Hostname, and have you tried checking the logs for services like Nginx to ensure there are no issues there?

So what is the best Practices for MSSP Model for customers whose working with Dynamic IPs I need to make sure that sensors can connect to public manager using any public IP. I tried to connect sensors over VPN and P2P network but it failed badly

For the web connection method it have a DDNS name for now to test it out

**Reply by weslambert** [reply]
We have many users that use VPN all the time. That would be best practice.  I wouldn't recommend having your instance directly accessible via the internet if you don't; have to.

**Reply by g33kroid** [reply]
Any specific VPN OpenVPN or Wire Guard that Security onion is compatible with

**Reply by chromefinch** [reply]
you could stand up a Kasm or Apache Guacamole instance that has access to it. that way you use a proxy and a jumphost with mfa to access something not meant to be exposed to the internet.
